clonevirtualenv.py

```diff
@@ -10,7 +11,7 @@
 import sys
 import itertools
 
-version_info = (0, 2, 6)
+version_info = (0, 2, 7)
```

Comment:
Bumping the version should be a separate commit that we do when releasing; can you remove this?

Reply:
Can do, but this is the wrong way of doing it - IMHO. The very first change I normally do to a module/package after deploying, is to bump the version number. The second after you have deployed you should never refer to the any new changes as a continuation of the old version. Changing the version should be the FIRST change you do, not the last...

Reply:
Yeah, I don't disagree (ideally, actually, it'd be something between 0.2.6 and 0.2.7 for the Git versions).  But I only just got involved in helping maintain this the last couple weeks, and regardless, it should be its own commit.